ml-agents/mlagents/trainers/stats.py

```diff
@@ -113,19 +112,7 @@ def write_stats(
             )
             if self.self_play and "Self-play/ELO" in values:
                 elo_stats = values["Self-play/ELO"]
-                mean_opponent_elo = values["Self-play/Mean Opponent ELO"]
-                std_opponent_elo = values["Self-play/Std Opponent ELO"]
```

Comment:
Do we still want this info somewhere?

Reply:
I'm not sure how valuable it is honestly. I don't think we need it.